Lesson5

VictorCher opened the pull request.

VictorCher committed:
Update Startup.cs

VictorCher committed:
Не работает tag-helper

VictorCher committed:
Починил tag-helper

VictorCher committed:
Сделал все задания по ДЗ №3

VictorCher committed:
Добавил странички

VictorCher committed:
Поправил ссылки на картинки в блоге

VictorCher committed:
Начал работу над ДЗ №4

VictorCher committed:
Почти все сделал, остались только фильтры

VictorCher committed:
Доделал, но ничего не работает

VictorCher committed:
Вроде всё заработало

VictorCher committed:
Сделал всё по методичке, но приложение пока не запускается

VictorCher committed:
Заработало, но данные (категории и бренды) не отображаются

kiruhin commented on WebStore/WebStore/Date/DbInitializer.cs:
```diff
@@ -22,9 +22,9 @@ public static void Initialize(WebStoreContext context)
+                context.Database.ExecuteSqlCommand("SET IDENTITY_INSERT [dbo].[Sections] OFF");
                 trans.Commit();
             }
             var brands = new List<Brand>();
```
списки пустые, поэтому и данных в БД нет, и не отображается ничего. надо скопировать списки из InMemoryProductData

kiruhin commented on WebStore/WebStore/Date/DbInitializer.cs:
```diff
@@ -2,7 +2,7 @@
 using System.Linq;
```
и в целом этот файл логичнее в проекте DAL хранить

kiruhin reviewed (approved):
списки доделать, тогда заработает =)

VictorCher committed:
Для магазина всё заработало.
Подключил сотрудников к БД , но не знаю что делать с интерфейсом IEmploeesData (если сделаю без EmployeesViewModel, то не смогу работать с формой)

VictorCher committed:
Компилируется, но не запускается

VictorCher committed:
Попытался сделать работу с пользователями через БД

kiruhin commented on WebStore/WebStore/appsettings.json:
```diff
@@ -1,7 +1,8 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.\\SQLEXPRESS;Initial Catalog=WebStore;Trusted_Connection=True;Connection Timeout=30;"
+    "DefaultConnection": "Server=SL-SERVISA;Initial Catalog=WebStore;Trusted_Connection=True;Connection Timeout=30;",
```
убедись, что имя сервера правильное. сходи в sql management studio и проверь подключение там
